bash-preexec.sh

```diff
@@ -199,14 +200,18 @@ __bp_preexec_invoke_exec() {
         if type -t "$preexec_function" 1>/dev/null; then
             __bp_set_ret_value $__bp_last_ret_value
             $preexec_function "$this_command"
-            preexec_ret_value="$?"
+            preexec_function_ret_value="$?"
+            if (( $preexec_function_ret_value != 0 )); then
```

Comment:
@jombooth I think your point is valid! I would have merged :) This way is more consistent I guess.